app/src/main/java/com/shevchenkovtwo/homework/MoviesAdapter.kt

```diff
@@ -46,4 +44,13 @@ class MoviesAdapter(private var movies: List<Movie>) :
     override fun getItemCount(): Int {
         return movies.size
     }
-}
\ No newline at end of file
+
```

Comment:
`selectedMovie` = `movies[position]`
`it.findNavController().navigate(R.id.action_fragmentMoviesList_to_movieDetailFragment)`
in dynamic lists it creats bug, in my Rick&Morty App already fix it